main.sh

```diff
@@ -7,6 +7,6 @@ while :
 do
   read -r question
   question="$(echo $question | tr -d \?)"
-  echo "The answer to $question is Pascal"
+  echo "The answer to $question is Matas"
```

Comment:
Have you tested this still outputs `Pascal`?... I'm not expert but I expect this might output `Matas` which is wrong. I'd be willing to see something along the following lines for clarification:


```suggestion
  echo "The answer to $question is Pascal and not Matas"
```